daml-lf/spec/daml-lf-1.rst

```diff
@@ -232,6 +228,13 @@ Version: 1.dev
 
   * **Add** intern package IDs in external package references.
 
+Version: 1.dev
+..............
+
+  * Last amendment date:
```

Comment:
Is that actually useful information?

Reply:
I want the place holder to be there for future update of `1.dev`. 

I added the following description, I found in the git history:
 
> * Description:
>
>    * **Change** nothing yet.

Reply:
I was not specific enough. Sorry. I'm wondering if the last amendment date is useful information?

Reply:
I dropped the line.